packages/financial-templates-lib/src/clients/FinancialContractEventClient.ts

```diff
@@ -241,98 +281,88 @@ class FinancialContractEventClient {
       this.newSponsorEvents.push({
         transactionHash: event.transactionHash,
         blockNumber: event.blockNumber,
-        sponsor: event.returnValues.sponsor,
+        ...event.returnValues,
```

Comment:
this will add a few more props to this object but I dont think it matters.

Reply:
Yeah, the tests actually depend on the exact members, so I'm going to update them to be more lenient as this is much more convenient to keep the types consistent.